Actions/AL-Go-Helper.ps1

```diff
@@ -1781,3 +1814,40 @@ Function AnalyzeProjectDependencies {
         $no++
     }
 }
+
+function GetBaseFolder {
+    Param(
+        [string] $folder
+    )
+
+    if (!(Test-Path (Join-Path $folder '.github') -PathType Container)) {
+        $folder = (Get-Item -Path $folder).Parent.FullName
```

Comment:
Wasn't there an issue with `Parent` in linux?

Reply:
The problem i discovered in Linux was with Get-ChildItem - but it might have been something else.
The above code works fine